includes/template.php

```diff
@@ -32,41 +32,20 @@
 // Add the `frontity_embedded` query.
 $url .= ( wp_parse_url( $url, PHP_URL_QUERY ) ? '&' : '?' ) . 'frontity_embedded=true';
 
-// Add a token to the URL if the current page is a preview, but only if a user
-// is logged in and can edit the post.
+// Get the entity ID.
 $id = get_the_ID();
```

Comment:
This might be an ignorant question: I see that https://developer.wordpress.org/reference/functions/get_the_id/ can return false if there is no "current item". Could it happen in this case and do we need to handle it?

The same goes for the `get_post_type()` function that is called below.

Reply:
Good point, although I think it's fine the way it's coded right now 🙂 

```php
// Get ID, could be false.
$id = get_the_ID();

// Continue if ID is not false (and other conditions).
if ( $id && ... ) {
  // Get the type. If ID is not false, the post exists and so does the type.
  $type = get_post_type();
}
```

Reply:
Oh, not sure how I missed this 😅 Thanks Luis!